torch/_inductor/compile_fx.py

```diff
@@ -381,6 +381,21 @@ def fake_tensor_prop(
     return fake_mode
 
 
+def should_use_remote_fx_graph_cache():
+    if config.fx_graph_remote_cache:
+        return True
+    if not config.is_fbcode():
+        return False
+    if torch.version.hip is not None:
+        return False
+
+    from triton.runtime.fb_memcache import MEMCACHE_VERSION
+
+    return MEMCACHE_VERSION >= torch._utils_internal.justknobs_getval_int(
+        "pytorch/remote_cache:pytorch/remote_cache:fx_graph_memcache_version"
+    )
```

Comment:
There's a hard coded version of memcache in code, so that we can do versioning. If we fix something, there might still be older training platforms around, so a boolean flag is not enough.